src/operation/operation.js

```diff
@@ -101,10 +101,10 @@ export default class Operation {
     /* Build an OpenAPI definition for this operation.
      */
     build(openapiVersion) {
-        return buildVersion(this, openapiVersion);
+        return pickVersion(this, 'build', openapiVersion)();
```

Comment:
All of the version picking code paths now specify the name and version of the function they way and get back the func. In most cases, we immediately turn around and call the same function.